src/App.js

```diff
@@ -31,9 +35,19 @@ function App() {
               <Link to="/">Bobcat Search</Link>
             </li>
             <li className="icon">
-              <NavLink to="/schedule">
-                <img src="./img/edit-calendar.svg" alt="Edit Calendar" />
-              </NavLink>
+              {
+                toggle !== "/schedule" ?
+                (
+                  <NavLink to={toggle} onClick={() => { console.log("true and ", toggle); setToggle("/schedule"); }}>
```

Comment:
We can remove the Debug messages for this line and line 46. as well as the brackets around the function bodies.